### PR TITLE
New blog post: P3 Quality

### PR DESCRIPTION
**Pull request by rachelcarmena**
New blog post about quality, focused on: People + Product + Process

### CODE REVIEW

File: _posts/2017-03-13-P3-Quality.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+date: 2017-03-13 11:00:00 +00:00
+author: Raquel M Carmena
+image:
+  src: /assets/img/custom/blog/2017-03-13-P3-Quality.jpg
+tags:
+- craftsmanship
+- quality
+
+---
+Do we know what is software quality? Is it measurable? If we want to improve the quality, do we know where to focus on?
```
Comment (sjn88) [inline _posts/2017-03-13-P3-Quality.md]: "do we know what to focus on" or "do we know where to focus"

File: _posts/2017-03-13-P3-Quality.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+A lot of agile adoptions fail, because they only focus on the process, not on technical excellence.
+
+That’s the reason why we need a global concept of quality, that we’ll call **P3 Quality**. If we want to ensure or improve quality, we should focus on:
+
+* People
+* Product
+* Process
+
+where the most important part is _people_ and others are around us:
```
Comment (sjn88) [inline _posts/2017-03-13-P3-Quality.md]: This sentence doesn't make sense. "and others are around us" Are you talking about the people, or about the quality metrics?

File: _posts/2017-03-13-P3-Quality.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+<center>
+<img src="{{site.baseurl}}/assets/img/custom/blog/2017-03-13-P3-Quality/P3-Quality.jpg" alt="P3 Quality" class="img img-responsive" width="70%"/>
+<small>*P3 Quality scheme*</small>
+</center>
+
+That’s the reason of **Software Craftsmanship**.
+
+When you read the book <a href="https://www.goodreads.com/book/show/23215733-the-software-craftsman" target="_blank">“The Software Craftsman. Professionalism, Pragmatism, Pride” by Sandro Mancuso</a>, you notice that the problems in software projects are the same, regardless of the company or the country, and the key to success is in the people’s quality, in our skills and attitude. That’s the importance in embracing Software Craftsmanship, always trying to improve all around us, included ourselves.
```
Comment (sjn88) [inline _posts/2017-03-13-P3-Quality.md]: "when" seems wrong here. Also "always trying to improve all around us" doesn't quite make sense. "included ourselves" should be "including ourselves".

Comment (rachelcarmena) [reply _posts/2017-03-13-P3-Quality.md]: Thanks @Seanseviltwin !!

**Comment by rachelcarmena** [comment]
I'm going to review this post with my mentor.

**Pull request closed without merging**